scenario/ncf_tests.py

```diff
@@ -15,13 +16,22 @@
 tag = get_param("tag", "")
 # Get CFEngine version used for the tests
 cfengine_version = get_param("cfengine_version", "")
+export_prefix = ""
+try:
+  download_user = os.environ['DOWNLOAD_USER']
+  download_password = os.environ['DOWNLOAD_PASSWORD']
+
+  if (download_user and download_password):
+    export_prefix = "export DOWNLOAD_USER=" + download_user + " DOWNLOAD_PASSWORD=" + download_password
+except:
+  export_prefix = ""
 
 # Get setup_ncf
 shell_on("agent", "wget -O /tmp/ncf-setup https://www.rudder-project.org/tools/ncf-setup", live_output=True)
 # Call setup_ncf
 if not cfengine_version:
   shell_on("agent", "sh /tmp/ncf-setup test-local https://github.com/Normation/ncf.git#branches/rudder/" + tag + " rudder-" + tag, live_output=True)
```

Comment:
if not cfengine version, means we use rudder, no ?
So we neeed it here but not below